packages/common-ethereum/src/project/models.ts

```diff
@@ -205,4 +202,6 @@ export class CustomDataSourceBase<K extends string, M extends SubqlMapping = Sub
   @Type(() => FileReferenceImpl)
   @IsObject()
   processor: FileReference;
+  @IsOptional()
+  options?: any;
```

Comment:
We should use the same type here.

https://github.com/subquery/datasource-processors/blob/96606d2563468d963966ccef030d43cb9da1e002/packages/ethermint-evm/src/index.ts#L92